devops/scripts/deploy-decoupled-upstream.sh

```diff
@@ -88,19 +100,24 @@ cp /tmp/decoupled-README.md ./README.md
 git add .
 
 echo "Committing changes"
-git commit -F /tmp/commit_message --author='Pantheon Automation <bot@getpantheon.com>'
+# TODO: UNCOMMENT
+#git commit -F /tmp/commit_message --author='Pantheon Automation <bot@getpantheon.com>'
```

Comment:
Committing is probably fine if we aren't pushing. The environment should get destroyed between builds. 
🔢